README.md

```diff
@@ -22,7 +22,7 @@ pod 'AsyncDisplayKit'
 
 (ASDK can also be used as a regular static library:  Copy the project to your
 codebase manually, adding `AsyncDisplayKit.xcodeproj` to your workspace.  Add
-`libAsyncDisplayKit.a`, AssetsLibrary, and Photos to the "Link Binary With
+`libAsyncDisplayKit.a`, MapKit, AssetsLibrary, and Photos to the "Link Binary With
```

Comment:
I think it's accurate to say that you can optionally add MapKit & Photos. From the pod spec AssetsLibrary is still a 'required' framework, so not sure about that one.